sys/riscv/riscv/pmap.c

```diff
@@ -648,11 +648,14 @@ pmap_bootstrap(vm_pointer_t l1pt, vm_paddr_t kernstart, vm_size_t kernlen)
 	/* Sanity check the index, KERNBASE should be the first VA */
 	KASSERT(l2_slot == 0, ("The L2 index is non-zero"));
 
-	freemempos = roundup2(KERNBASE + kernlen, PAGE_SIZE);
+	freemempos = KERNBASE;
```

Comment:
Probably `freemempos` is not the right name for this, it should be renamed upstream to `kernel_space` or something.